pkg/config/config.go

```diff
@@ -182,6 +182,13 @@ type Config struct {
 			// Won't run when is equal or less than 0.
 			HealthCheckInterval time.Duration `mapstructure:"health_check_interval"`
 		} `mapstructure:"dynamodb"`
+
+		CosmosDB *struct {
+			ReadWriteKey SecureString `mapstructure:"readwritekey"`
```

Comment:
- Changed to `read_write_key`.
- Since at the same place you're getting that key, you can get `read_key` just as easily. It's confusing so I prefer to be explicit.